Onboarding/engineering-best-practices.md

```diff
@@ -51,6 +51,14 @@ Here is a brief list of Veteran Services Platform (VSP) recommended engineering
 - If modifying an existing app, we encourage writing code that's [easy to delete](https://programmingisterrible.com/post/139222674273/write-code-that-is-easy-to-delete-not-easy-to). We encourage coarse feature gating logic (i.e. "please repeat yourself") to avoid needing "shotgun surgery" later when the feature goes live. 
 - We rely heavily on automated testing in our continuous integration flow to detect regressions. It's important to follow our unit testing and end to end testing patterns to maintain a highly collaborative environment. 
 
+### 508/Accessibility 
```

Comment:
I think this section is actually much more detailed than the rest of the surrounding stuff --- I don't want to lose this information, but this reads more like a checklist of things to do rather than high-level guidance to remember.

maybe separate this out into a separate accessibility doc (if we don't have this already, but I kinda suspect we do) and add a bullet point to "Frontend" that's basically "make sure to accessibility test all UI changes, we have [documentation to help you do that the right way] (link)" ?